validation/schemas/qv1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://virtual.labs/schemas/qv1.json",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "question": {
+        "type": "string"
+      },
+      "answers": {
+        "type": "object",
+        "minProperties": 1,
```

Comment:
I'd think minimum no of answers should be 2, at least one incorrect. Do you see a use case for only the correct answer?